src/saveload/marker.rs

```diff
@@ -357,57 +363,70 @@ pub trait MarkerAllocator<M: Marker>: Resource {
     fn maintain(&mut self, _entities: &EntitiesRes, _storage: &ReadStorage<M>);
 }
 
-/// Basic marker implementation usable for saving and loading
-#[derive(Clone, Copy, Debug, Hash, PartialEq, Eq, PartialOrd, Ord, Serialize, Deserialize)]
-pub struct U64Marker(u64);
+/// Basic marker implementation usable for saving and loading, uses `u64` as identifier
+#[derive(Derivative, Serialize, Deserialize)]
+#[derivative(Clone, Copy, Debug, Hash, PartialEq, Eq)]
+pub struct SimpleMarker<T: ?Sized>(u64, PhantomData<T>);
```

Comment:
I think the second field should be skipped from the serialization. Maybe you could even make it transparent so we don't have a single-field tuple and just a `u64`.